UnitsNet.Tests/CustomCode/EnergyTests.cs

```diff
@@ -37,6 +38,7 @@ public class EnergyTests : EnergyTestsBase
         protected override double ErgsInOneJoule => 10000000;
 
         protected override double MillijoulesInOneJoule => 1000;
+        protected override double TeraelectronVoltsInOneJoule => 6241509.34326018;
```

Comment:
Are there different values for this? I see Kilo, Mega, Giga and Tera all have different digits, while I expected only the power value to be different (1e3, 1e6, 1e9 etc).

Also, as an example, your value of `TeraelectronVoltsInOneJoule  => 6241509.34326018` is different than `6241506.479963235` from https://www.convertunits.com/from/TeV/to/joule.

I'd like these to be consistent, if there exists one accurate constant to use for all of them.

Reply:
`6.241509343260179e18` is the constant used for `ElectronVoltsInOneJoule`. Unless this one is wrong too, I suggest using it as basis for all the others.
Also, please use scientific notation 1.12345678e15 for all these numbers so it is easier to read for large/small numbers.

Reply:
Is that better? :(
Or you need to record explicit results?